source/js/main.js

```diff
@@ -146,6 +146,64 @@ let main = {
       MozfestCarousels.init();
       MozfestHeroCarousels.init();
     }
+
+    const today = new Date();
+
+    // skip the banner if it got dismissed by the user today already
+    if (parseInt(localStorage.getItem("DISMISS_KEY")) === today.getDay()) {
```

Comment:
let's make this a const called `DISMISS_KEY` poiting to the actual text `donate banner dismiss day`, so that it's properly informative about what value we put in localStorage. That way, people can make informed decisions when looking at stored site data.